README

```diff
@@ -22,7 +22,10 @@ In either case, you should bump the version numer in debian/changelog.
 
 To build this package, check out clearwater-debian and run:
 
-`git-buildpackage -uc -us --git-ignore-branch`
+```
+./config/autorun.sh
+git-buildpackage -uc -us --git-ignore-branch --git-ignore-new
```

Comment:
I would split this out so that the default is still `git-buildpackage -uc -us --git-ignore-branch`, but you then have a sentence like "To build the debian when you have changes you don't want to commit yet, add `--git-ignore-new`"